### DISCUSSION TITLE
synology Unable to start

### DISCUSSION THREAD

**Discussion by xiaoleigithub** [containrrr/watchtower#1054] [Q&A]
synology dsm7.0

ERRO[0001] Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
<img width="1532" alt="截屏2021-08-13 08 51 52" src="https://user-images.githubusercontent.com/46390738/129287909-883a1446-a97e-4d31-8e0f-95c9b233aa6b.png">

**Comment by simonaronsson** [comment]
What command are you using to launch watchtower? You have to mount `/var/run/docker.sock` in the container for watchtower to work.

**Reply by xiaoleigithub** [reply]
我使用的是群晖安装的docker
